src/android/SplashScreen.java

```diff
@@ -194,9 +263,25 @@ public void run() {
                 // TODO: Use the background color of the webView's parent instead of using the
                 // preference.
                 root.setBackgroundColor(preferences.getInteger("backgroundColor", Color.BLACK));
+                
                 root.setLayoutParams(new LinearLayout.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT,
                         ViewGroup.LayoutParams.MATCH_PARENT, 0.0F));
-                root.setBackgroundResource(drawableId);
+                
+                // Use an ImageView to render the image because of its flexible scaling options.
+                splashImageView = new ImageView(context);
+                splashImageView.setImageResource(drawableId);
```

Comment:
I added a loop in JS to slow down the app. It calls console.log a bunch of times. I hard-coded BG color to RED and tested the app with and without the linear layout. I never see RED in either view hierarchy. When the app is slow, I just see black background when layout/painting is lagging.

![image](https://cloud.githubusercontent.com/assets/1012600/6909507/ad513bf8-d6fb-11e4-9146-e3f46a151912.png)